mandos-go/json/write/writeCommon.go

```diff
@@ -54,10 +54,10 @@ func checkAccountsToOJ(checkAccounts *mj.CheckAccounts) oj.OJsonObject {
 		if len(checkAccount.Comment) > 0 {
 			acctOJ.Put("comment", stringToOJ(checkAccount.Comment))
 		}
-		if !checkAccount.Nonce.IsDefault() {
+		if !checkAccount.Nonce.IsUnspecified() {
```

Comment:
Might be cleaner to have the positive alternative `if IsSpecified()` rather than the non-negative `if not IsUnspecified()`.